### ISSUE TITLE
mypy reports error: Skipping analyzing "fxpmath": module is installed, but missing library stubs or py.typed marker

### ISSUE THREAD

**Issue by jtmoon79** [francof2a/fxpmath#94]
***tl;dr*** `mypy` reports error when reviewing a file with `import fxpmath`:
`error: Skipping analyzing "fxpmath": module is installed, but missing library stubs or py.typed marker  [import-untyped]`

## Problem

Running _mypy_ against a Python file that imports `fxpmath` fails with error
```
PS> mypy .\testA.py
testA.py:1: error: Skipping analyzing "fxpmath": module is installed, but missing library stubs or py.typed marker  [import-untyped]
testA.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

### Reproduction

Using Python 3.8.10

Create a simple Python file `testA.py`

```
import fxpmath


if __name__ == "__main__":
    print(str(dir(fxpmath)))
```

Run `mypy` against the simple Python file `testA.py`. Use a virtual environment.

```
PS> python -m virtualenv --copies .virtualenv
PS> .\.virtualenv\Scripts\activate.ps1
PS> pip install fxpmath mypy
PS> mypy .\testA.py
testA.py:1: error: Skipping analyzing "fxpmath": module is installed, but missing library stubs or py.typed marker  [import-untyped]
testA.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports 
```

This installed `fxpmath` version 0.4.9, `mypy` version 1.8.0.

This bug was tested on Windows 10. This bug should occur on every Python platform.

## Solution

One of:

- Provide py.typed markers in `fxpmath` as suggested by mypy
- Provide a `types-fxpmath` types stub package (similar to, for example, [`types-setuptools`](https://pypi.org/project/types-setuptools/)).

**Comment by francof2a** [comment]
Thanks for the clear report and reproduction. We confirmed the issue: `fxpmath` was missing PEP 561 typing metadata, so mypy treated it as untyped and emitted:

`Skipping analyzing "fxpmath"... missing library stubs or py.typed marker [import-untyped]`

Fix implemented:
1. Added `fxpmath/py.typed`
2. Updated packaging metadata to include it in distributions (`pyproject.toml` package-data)

Validation:
1. Reproduced the error in a clean virtual environment before the change
2. Reinstalled patched package
3. Re-ran mypy on a minimal `import fxpmath` script
4. Result: `Success: no issues found in 1 source file`

Also added regression coverage (`test_issue_94_v0_4_9`) to ensure the marker is present and declared in packaging config, so this does not regress.

Note: this fix addresses the *import-untyped* error. It does not automatically provide rich types for all APIs; that requires adding annotations/stubs incrementally over time.

**Comment by francof2a** [comment]
Solved in version `0.4.10`
